WhereIsMyMouse/PluginUI.cs

```diff
@@ -92,7 +93,20 @@ public void CursorAura()
                 }
             }
 
-
+            if (Rainbow)
+            {
+                ImGui.ColorConvertRGBtoHSV(color.X, color.Y, color.Z, out float h, out float s, out float v);
+                if (h >= 1)
+                {
+                    h = 0;
+                }
+                h += cycleSpeed;
```

Comment:
noticed this a bit late but this will make it cycle at a different speed depending on framerate; consider using `ImGui.GetIO().DeltaTime` instead.

Reply:
Good idea 👍
I changed it to DeltaTime on my fork. Is there a way to reopen the PR or should I make a new one?

Reply:
Sadly you can't reopen a merged PR, making a new one is okay👌